Automattic/wp-calypso
# Components: DropZone: Add Redux actions for show/hide events

This PR is a part of ongoing improvements before merging #11839. It adds a couple of Redux actions that will get fired when a DropZone is shown or hidden so the components may show/hide elements or generally take an action in such event.

Right now the PR should not have any side effects.

To test:
1. Checkout branch or use Calypso.live
2. Go to the editor
3. Verify that dragging/dropping image on the editor works properly
4. Verify dragging/dropping image in the Media Library works properly ( Access it through the `+` icon in the editor )
5. Verify the post can be saved/updated properly
6. Verify there are no errors in the JS console.
